portfolio/src/main/webapp/js/scripts.js

```diff
@@ -0,0 +1,42 @@
+/*!
+    * Start Bootstrap - Resume v6.0.0 (https://startbootstrap.com/template-overviews/resume)
+    * Copyright 2013-2020 Start Bootstrap
+    * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap-resume/blob/master/LICENSE)
+    */
+    (function ($) {
+    "use strict"; // Start of use strict
+
+    // Smooth scrolling using jQuery easing
+    $('a.js-scroll-trigger[href*="#"]:not([href="#"])').click(function () {
+        if (
+            location.pathname.replace(/^\//, "") ==
+                this.pathname.replace(/^\//, "") &&
+            location.hostname == this.hostname
+        ) {
+            var target = $(this.hash);
```

Comment:
prefer const and let over var (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)

Reply:
done, thanks!